### PR TITLE
Increase test coverage to 99.44%

### PR DESCRIPTION
**Pull request by edgarsilva**

### CODE REVIEW

File: lib/adaptors/serialport.js [hunk 1]
```diff
@@ -119,3 +119,7 @@ Adaptor.prototype.onRead = function read(callback) {
 Adaptor.prototype.close = function close(callback) {
   this.serialport.close(callback);
 };
+
+Adaptor.prototype.newSerialport = function() {
+  return new SerialPort(this.conn, {});
+};
```
Comment (stewart) [inline lib/adaptors/serialport.js]: We really don't need this level of abstraction. Let the test coverage score suffer a little bit, I'd rather have that than meaningless redirects in the code.

Comment (stewart) [inline lib/adaptors/serialport.js]: And in this case you could just re-write to stub out `require("thing").Serialport`

Comment (edgarsilva) [inline lib/adaptors/serialport.js]: Agreed, will de-abstract that. What do you mean by stub require? e.g.?

Comment (stewart) [inline lib/adaptors/serialport.js]: The serial port constructor isn't a bare require - so you should be able to 

```javascript
var serialport = require("serialport");
stub(serialport, "SerialPort");
```

Comment (edgarsilva) [inline lib/adaptors/serialport.js]: word, will update and squash. What about adaptor and loader?

Comment (stewart) [inline lib/adaptors/serialport.js]: I'll take a look later, not too worried for now.

Comment (edgarsilva) [inline lib/adaptors/serialport.js]: @stewart I can't make that test pass as you suggested, if you want to try to fix it as suggested you are welcome. The only way I could make them past is as they are right now.

**Comment by deadprogram** [comment]
:+1:

**Pull request merged**